### DISCUSSION TITLE
WPF League of Legends

### DISCUSSION THREAD

**Discussion by jamesnet214** [dotnet/wpf#8462] [Show and tell]
Hello, I am James, a WPF developer and a Microsoft MVP from Korea. I would like to share about the League of Legends project, which I open-sourced a few years ago to showcase the excellence of WPF and the flexibility of XAML. I hope you find it interesting.

This project is implemented in .NET 7.0 and can be built using Visual Studio 2022 or later.

https://github.com/jamesnet214/leagueoflegends

![image](https://github.com/dotnet/wpf/assets/52397976/6603cd36-e2b3-4396-82f9-9f78f1005d9d)

**Comment by lindexi** [comment]
Awesome @jamesnet214

**Reply by lindexi** [reply]
And I found the Video about this in : https://b23.tv/gMUrb7V

**Reply by jamesnet214** [reply]
@lindexi Oh, thank you so much!! And the video you found is one made by my wife. It's an honor that you know about our video. Also, in a few days, we plan to upload an English version through YouTube. 😄

**Reply by lindexi** [reply]
Haha, Say hello to her for me —— Greetings from her fellow Chinese.

**Reply by vickyqu115** [reply]
Hello! I'm Vicky, and it's a great honor to be greeting you like this. My hometown is Nanjing, and I'm currently in Seoul, South Korea. I hope we can continue to interact and exchange ideas. I'm genuinely happy to meet you! 😄

**Reply by singhashish-wpf** [reply]
@jamesnet214 안녕하세요. 공유해 주셔서 감사합니다 :)
Hello and Thank you for sharing this :)

**Reply by jamesnet214** [reply]
@singhashish-wpf I was surprised and really moved to see you responding in Korean. I'm truly grateful to everyone for viewing it positively, and I am really happy.😆😆
